sklearn_numba_dpex/kmeans/drivers.py

```diff
@@ -131,12 +133,12 @@ def __init__(
             or device_params.preferred_work_group_size_multiple
         )
 
+        # So far the best default value has been empirically found to be 2 so we use
+        # this default.
+        # TODO: when it's available in dpctl, use the `max_group_size` attribute
+        # exposed by the kernel instead ?
```

Comment:
IIRC, `numba_dpex` has a function to get the optimal values for `work_group_size` and similar variables; is this right, @oleksandr-pavlyk?